.release/ci.hcl

```diff
@@ -0,0 +1,92 @@
+schema = "1"
+
+project "terraform-ls" {
+  team = "terraform-editor-experience"
```

Comment:
This attribute is document as unused - is there a reason to include it at this point? If so - what does it refer to specifically?

Reply:
FWIW I omitted it for `hc-install` and the initial CI run was green, although I didn't get to staging/promotion due to other reasons yet.